management/schemas/campaign.yaml

```diff
@@ -62,21 +62,15 @@ schemas:
       CustomFieldsJson:
         type: string
         nullable: true
-      EndDate:
-        type: string
-        nullable: true
-        deprecated: true
```

Comment:
I think this is a valid param for a `/v2/campaign` request. Same as `StartDate`, that is why I moved it from `Campaign` to `CampaignBase` on my [PR](https://github.com/adzerk/adzerk-api-specification/pull/74).

Reply:
@tomascasas Hmm, I don't think that's right. StartDate and EndDate should not be a valid attribute for any campaign create request (neither v1 nor v2). Both of those values are calculated based on the flights inside the campaign and so they should only be included in responses, not in requests.

Reply:
You are 100% right, I just checked docs and even our own implementation and validated with POSTMAN that is correct. Sorry for the confusion.